### DISCUSSION TITLE
NFL feature

### DISCUSSION THREAD

**Discussion by evanmolinelli** [georgedouzas/sports-betting#65] [Ideas]
Hi,

I'm starting to create a new feature to analyze NFL data. Forgive me for I am new to machine learning but I've been a fan of the NFL and sports gambling for a while as well as a software engineering background. Here is my first pull request and would love some feedback/help/recommendations/criticism.
https://github.com/evanmolinelli/sports-betting/pull/1

**Comment by georgedouzas** [comment]
Hi @evanmolinelli,

Could you do a PR to the current repo? Then we will continue the discussion there.

**Comment by evanmolinelli** [comment]
@georgedouzas I am unable to push to the repo. 
`git push origin nfl-data
remote: Permission to georgedouzas/sports-betting.git denied to evanmolinelli.
fatal: unable to access 'https://github.com/georgedouzas/sports-betting.git/': The requested URL returned error: 403`

**Reply by georgedouzas** [reply]
Please read [this](https://www.asmeurer.com/git-workflow/).

**Comment by evanmolinelli** [comment]
Thank you for the reference. Just created the new PR for the original repo.
